specs/reference/data-services/security.v1.json

```diff
@@ -0,0 +1,56 @@
+{
+  "openapi": "3.0.0",
+  "tags":[{
+    "name": "security"
```

Comment:
I know we have a "SecurityService" already (I may have been the one to suggest that name, can't remember), but I'm wondering if we should rename it to "UserService" because "getAuthorities" is specific to a user. I'm wondering what other functions we would add to this... and I'm not sure. So I don't really know a good name, but I lean towards UserService since this is currently specific to the user calling it. Thoughts?

Reply:
Makes sense. Leaves the door open for Hub Central to do some limited security management, etc. in the future.